lib/capturable.js

```diff
@@ -11,16 +11,23 @@ module.exports = function capturable () {
     function _expr (value, args) {
         var captured = events;
         events = [];
+        var source = {
+            content: args.content,
+            filepath: args.filepath,
+            line: args.line
+        };
+        if (args.generator) {
+            source.generator = true;
+        }
+        if (args.async) {
+            source.async = true;
+        }
```

Comment:
@jamestalmage Correct.

` _expr` function captures and collects values for each assertion argument.
So when assertion expression is `assert.equal(foo, bar)`, `_expr` is called twice (foo and bar).

- `source` is for information of assertion expression code itself, so `source` of two args are the same.
- `powerAssertContext` is for captured values and its paths, so `powerAssertContext` of two args are not same.

I agree that these variable names are confusing. I'll fix it when this pull-req lands.